src/js/plugins/plugin.iframe.js

```diff
@@ -0,0 +1,47 @@
+/**
+ * Plugin for two-way communication between a BookReader in an IFrame and the
+ * parent web page
+ */
+
+const MESSAGE_TYPE_FRAGMENT_CHANGE = 'bookReaderFragmentChange';
+
+BookReader.prototype.init = (function(super_) {
+    return function() {
+        super_.call(this);
+        _attachEventListeners(this);
+    };
+})(BookReader.prototype.init);
+
+/**
+ * @private
+ * Using window.postMessage() and event listeners, the plugin notifies the
+ * parent window when pages change, and the parent window can also
+ * explicitly request a page change by sending its own message.
+ *
+ * @param {BookReader} br
+ * @param {Window?} [parent]
+ */
+export function _attachEventListeners(br, parent=window.parent) {
```

Comment:
How come you prefixed this with an underscore?

Reply:
I'm only exporting this because I want to test it :P It actually shouldn't be exported. I'm opting to use the underscore to better communicate that it's private. Not crazy about this, but I'm not sure if there's a better way.